test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,18 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
You don't need to include `throws Exception` here. Nevertheless, it would be a good exercise to find out the difference between assertion and exception :)